.github/workflows/publish_connectors.yml

```diff
@@ -64,6 +64,13 @@ jobs:
         shell: bash
         run: ./poe-tasks/get-modified-connectors.sh --prev-commit --json  --java | ./poe-tasks/build-and-publish-java-connectors-with-tag.sh --main-release --publish
 
+      # If you modify this step, remember to also modify publish-connector-metadata-manual.
```

Comment:
is my understanding correct: this is ok because we are also removing this functionality out of airbyte-ci, so we don't risk double publishing?

Reply:
pretty much. I'm not even moving the metadata publish step yet, this PR only moves the metadata validation step. AFAICT the sequence was previously:
* in GHA:
    * publish JVM images
* in airbyte-ci:
    * validate metadata
    * publish image if not exists (i.e. python/manifest-only)
    * push metadata, spec, etc.

after this PR, it will be:
* in GHA:
    * publish JVM images
    * validate metadata
* in airbyte-ci:
    * publish image if not exists (i.e. python/manifest-only)
    * push metadata, spec, etc.

so the exact same steps, just invoked differently.

Reply:
Ah! I didn't realise this was validating connector metadata file pre-publish. I thought this was validating the metadata files **in Cloud**. I think it's useful to highlight this distinction somewhere.